In [1]:
from googleapi import *

In [2]:
place = find_aggregated_places()


! ERROR: 429


In [3]:
print(place['error']['message'])

The response exceeded the maximum number of allowed places. Only 100 places can be returned in a single response. The request produced 2648 places and therefore no places were returned.


> The response exceeded the maximum number of allowed places. Only 100 places can be returned in a single response. The request produced **2648** places and therefore no places were returned.